--- packages/framework-arduinoteensy/cores/teensy4/yield.cpp	2023-12-11 17:38:48.495445219 +0800
+++ packages/framework-arduinoteensy/cores/teensy4/yield.cpp	2023-12-11 17:41:10.658063282 +0800
@@ -31,7 +31,7 @@
 #include <Arduino.h>
 #include "EventResponder.h"
 
-#ifdef USB_TRIPLE_SERIAL
+#if defined(USB_TRIPLE_SERIAL) || defined(USB_RAWHID_TRIPLESERIAL)
 uint8_t yield_active_check_flags = YIELD_CHECK_USB_SERIAL | YIELD_CHECK_USB_SERIALUSB1 | YIELD_CHECK_USB_SERIALUSB2; // default to check USB.
 extern const uint8_t _serialEventUSB2_default;	
 extern const uint8_t _serialEventUSB1_default;	
@@ -61,13 +61,13 @@
 		if (_serialEvent_default) yield_active_check_flags &= ~YIELD_CHECK_USB_SERIAL;
 	}
 
-#if defined(USB_DUAL_SERIAL) || defined(USB_TRIPLE_SERIAL)
+#if defined(USB_DUAL_SERIAL) || defined(USB_TRIPLE_SERIAL) || defined(USB_RAWHID_TRIPLESERIAL)
 	if (yield_active_check_flags & YIELD_CHECK_USB_SERIALUSB1) {
 		if (SerialUSB1.available()) serialEventUSB1();
 		if (_serialEventUSB1_default) yield_active_check_flags &= ~YIELD_CHECK_USB_SERIALUSB1;
 	}
 #endif
-#ifdef USB_TRIPLE_SERIAL
+#if defined(USB_TRIPLE_SERIAL) || defined(USB_RAWHID_TRIPLESERIAL)
 	if (yield_active_check_flags & YIELD_CHECK_USB_SERIALUSB2) {
 		if (SerialUSB2.available()) serialEventUSB2();
 		if (_serialEventUSB2_default) yield_active_check_flags &= ~YIELD_CHECK_USB_SERIALUSB2;
